python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -116,8 +117,11 @@ def build_robot_from_log(
         log_data: Dict[str, Any],
         mesh_path_dir: Optional[str] = None,
         mesh_package_dirs: Sequence[str] = (),
+        *, robot_name: str = ""
         ) -> jiminy.Robot:
-    """Build robot from log.
+    """Build robot from log. This function can only be used for logs of
```

Comment:
From what I seeing. It can actually be used for multi-robot simulations. Just that it can only load one specific robot, providing its name.